packages/create-atomic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@coveo/create-atomic",
+  "description": "Coveo Atomic Generator",
+  "version": "1.18.0",
+  "bin": {
+    "create-atomic": "./index.mjs"
+  },
+  "main": "index.mjs",
+  "scripts": {
+    "start": "plop",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
This will might make CI crash because the root test script invoke lerna exec test iirc